lib/java-javadoc.gradle

```diff
@@ -174,6 +174,39 @@ allprojects {
                 return success
             }
 
+            def addJdk11OfflineLink = { name, url, elementListFile ->
+                def modulePrefix = 'module:';
+                def inModule = false
+                def currentModule = ''
+                List<String> lines = []
+                elementListFile.eachLine { line ->
+                    if (line.startsWith(modulePrefix)) {
+                        if (inModule) {
+                            inModule = false;
+                            if (!lines.isEmpty()) {
+                                def javadocUrl = "${url}${currentModule}/"
+                                def javadocUrlSha1 = MessageDigest.getInstance('SHA1').digest(javadocUrl.getBytes('UTF-8')).encodeHex()
+                                def listFileDir = new File(javadocCacheDir, "${name}/${currentModule}/${javadocUrlSha1}")
+                                def moduleElementListFile = new File(listFileDir, 'element-list')
+                                moduleElementListFile.parentFile.mkdirs()
+                                moduleElementListFile.withWriter { out ->
+                                    lines.each {
+                                        out.println it
+                                    }
+                                }
+
+                                linksOffline javadocUrl, "${listFileDir}"
```

Comment:
Don't we need to clear the `lines` from here?

Reply:
Oops, fixed, Thanks~!